htdocs/categories/class/api_categories.class.php

```diff
@@ -265,6 +265,246 @@ public function delete($id)
         );
     }
 
+    /**
+     * Link an object to a category by id
+     *
+     * @param int $id  ID of category
+     * @param string   $type Type of category ('member', 'customer', 'supplier', 'product', 'contact')
+     * @param int      $object_id ID of object
+     *
+     * @return array
+     * @throws RestException
+     *
+     * @url POST {id}/objects/{type}/{object_id}
+     */
+    public function link_object_by_id($id, $type, $object_id)
```

Comment:
Can you use camelCaps style please? linkObjectById, linkObjectByRef,etc

Reply:
it is done. :)